readthedocs/organizations/migrations/0013_migrate_locked_subscriptions.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.4 on 2023-08-21 20:28
+
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    """Migrate locked subscriptions to never_disable organizations."""
+    Subscription = apps.get_model("subscriptions", "Subscription")
+    locked_subscriptions = Subscription.objects.filter(locked=True).select_related(
+        "organization"
+    )
+    for subscription in locked_subscriptions:
+        subscription.organization.never_disable = True
+        subscription.organization.save()
```

Comment:
We could just manually run this from the django shell, to avoid having a reference to the old Subscription mode, that will make it easier to delete the app later.

Reply:
I think this is fine for now. We should probably squash the migrations after running them and cleanup all the ones that are old.

Reply:
This migration is in the organizations model, so we would need to squash those in order to delete the subscriptions model. I'm inclined to just do it from the shell, thinking more about it.

Reply:
Please, add a "post-deploy" note for this with clear instructions, so we don't forget.

Reply:
Done